### ISSUE TITLE
Lives remaining count for each client does not get reset for new players

### ISSUE THREAD

**Issue by AGSPhoenix** [Daimyo21/Insurgency-dy-sourcemod#21]
Looks like there is already a system to track each player by SteamID and give them lives on their first join [here](https://github.com/Daimyo21/Insurgency-dy-sourcemod/blob/f49196c0dfd49ed62f4f93491f3b7672e0e07f26/scripting/c_dy_respawn_naong.sp#L3470), but it's not triggering for some reason. I haven't been able to get a development environment up and running to poke around and figure out why though, and nothing is obviously wrong with the code.

I'll probably take another shot at getting a testing server set up soon, but the lack of documentation makes it a real pain in the ass.

If you have any insights on either front, I'm all ears.

**Comment by thecannons** [comment]
Yeah AGSPhoenix, I tried to figure it out but Ill have to play with it another time.

**Comment by AGSPhoenix** [comment]
I was trying to get more information about this, and figured checking to see if the server log showed the new player join message would help. I tried to get Panda and Major Brown to take a look a few days ago, but they weren't really in the mood to go digging through server logs.

Do you think you could take a look? Should just be a quick search for `is new! | SteamID:`.

**Comment by ozzygit** [comment]
Actually I've noticed this in console too. It considers me a new player everytime!
